<?xml version="1.0"?>
<launch>
    <param name="/use_sim_time" value="true"/>
    <node name="laser_tools" pkg="laser_tools_src2" type="laser_tools.py" output="screen">
      <rosparam
        command="load"
        file="$(find laser_tools_src2)/config/scout_laser_tools.yaml"
        subst_value="true"
      />
    </node>
</launch>
